forgettable-frontend/src/pages/EncountersListPage/EncountersListPage.js

```diff
@@ -0,0 +1,221 @@
+import EncounterDrawer from '../../components/EncounterDrawer/EncounterDrawer';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import EncounterDetailsModal from '../../components/EncounterDetailsModal/EncounterDetailsModal';
+import {deleteEncounter, getAllEncounters, searchEncounter, getEncountersByPage} from '../../services';
+import CustomModal from '../../components/CustomModal/CustomModal';
+import {ToastContainer, toast} from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+export default function EncountersListPage() {
+  const PAGE_SIZE = 10;
+
+  const [pageNum, setPageNum] = useState(1);
+
+  const [isHover, setIsHover] = useState(false);
+  const [selectedInfo, setSelectedInfo] = useState(undefined);
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [hasMore, setHasMore] = useState(true);
+
+  const [selectedEncounter, setSelectedEncounter] = useState(undefined);
+  const [selectedEncounterId, setSelectedEncounterId] = useState('');
+  const [encounterModalOpen, setEncounterModalOpen] = useState(false);
+
+  const [deleteModalOpen, setDeleteModalOpen] = useState(false);
+
+  const [encounterList, setEncounterList] = useState([
+    {
+      _id: '',
+      title: '',
+      date: null,
+      location: '',
+      description: '',
+      persons: [{
+        _id: '',
+        first_name: '',
+        img: '',
+      }],
+    },
+  ]);
+
+  useEffect(async () => {
+    const result = await getEncountersByPage(pageNum, PAGE_SIZE);
+    setEncounterList(result);
+    setIsLoading(false);
+  }, []);
+
+  const handleCardClick = (encounter) => {
+    setSelectedEncounter(encounter);
+    setEncounterModalOpen(true);
+  };
+
+  const handleModalClose = () => {
+    setEncounterModalOpen(false);
+  };
+
+  const onDelete = (encounterId) => {
+    setSelectedEncounterId(encounterId);
+    setDeleteModalOpen(true);
+  };
+
+  const onDeleteConfirmed = async (encounterId) => {
+    const result = await deleteEncounter(encounterId);
+    if (result) {
+      const updatedEncountersList = await getAllEncounters();
+      setEncounterList(updatedEncountersList);
+      toast.success('Encounter deleted!', {
+        position: 'bottom-center',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+    } else {
+      toast.error('Something went wrong... :(', {
+        position: 'bottom-center',
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+    }
+    setDeleteModalOpen(false);
+    setEncounterModalOpen(false);
+  };
+
+  const fetchMoreData = () => {
+    if (!hasMore) {
+      return;
+    }
+
+    setTimeout(async () => {
+      const newPageNum = pageNum + 1;
+      const newResult = await getEncountersByPage(newPageNum, LIMIT);
```

Comment:
Noticing that there isn't a LIMIT variable, maybe you forgot to rename either LIMIT or PAGE_SIZE?

Reply:
Nevermind, you already got to it as I was reviewing haha

Reply:
Yea just updated it!